### PR TITLE
Moneta

### PR DESCRIPTION
**Pull request by MatthewTristan**
Hello,

Recently we had our PR approved, but our community members are mentioning a few things:

https://github.com/DefiLlama/DefiLlama-Adapters/pull/11946

1. The token should be in the stablecoin category, since its a full stablecoin backed 1:1 by the dollar, not any other non-dollar asset. How do I add Moneta/USDM to the stablecoin category? I was confused since the only two categories mentioned in the template are algo stablecoins and decentralized stablecoins, but this really is a pure stablecoin.

Basically, how can Moneta/USDM be listed on this page?

2. The audit is missing from the page, and the page says there has been no audit performed, even though I included the audit from Sundae labs

https://cdn.sundaeswap.finance/audits/mehen.pdf (Mehen Rebranded to Moneta, nothing in the smart contract has changed.)

### CODE REVIEW

**Comment by Define101** [comment]
@MatthewTristan is it good now ? 
image

**Comment by g1nt0ki** [comment]
add moneta to stablecoins here: https://github.com/DefiLlama/peggedassets-server/commit/f735546092c1910137e45847c8ebb92cc7a3f359

**Comment by MatthewTristan** [comment]
@Define101 @g1nt0ki Thank you both a ton!
